src/Roslyn.Suppressions/Roslyn.Suppressions/SupportedSuppressionBuilder.g.cs

```diff
@@ -493,6 +493,7 @@ private static ImmutableArray<SuppressionDescriptor> GetDescriptors()
                 CreateDescriptor("S6424"),
                 CreateDescriptor("S6444"),
                 CreateDescriptor("S6507"),
+                CreateDescriptor("S6513"),
```

Comment:
Did you have to do anything to regenerate this file, or did it happen automatically as part of the build? (it's supposed to be automatic now).

Reply:
I was meaning to ask you about this, it wasn't. I first ran the build in the IDE - not generated. Then I ran it via command line with msbuild - not generated. Finally I right-clicked on it to regenerate.